pkg/solidity-utils/contracts/helpers/BufferHelpers.sol

```diff
@@ -25,7 +25,10 @@ library BufferHelpers {
 
         uint256 wrappedBalanceAsUnderlying = 0;
         if (bufferBalance.getBalanceDerived() > 0) {
-            wrappedBalanceAsUnderlying = wrappedToken.convertToAssets(bufferBalance.getBalanceDerived());
+            // Buffer underlying surplus is used when wrapping (it means, deposit underlying and get wrapped tokens),
+            // so we use `previewMint` to convert wrapped balance to underlying. `mint` is the exact opposite
+            // operation as `deposit, used to wrap underlying tokens`
```

Comment:
```suggestion
            // operation as `deposit, used to wrap underlying tokens`.
```

Reply:
Consider replacing "`mint ` is the exact opposite operation as `deposit `, used to wrap underlying tokens."
with: "The mint function is used here, as it performs the inverse of a deposit operation." for consistency with the comments below (in `_wrapWithBuffer`).